src/controllers/breaches.js

```diff
@@ -189,13 +176,13 @@ async function bundleVerifiedEmails (options) {
     : []
 
   for (const breach of foundBreachesWithRecency) {
-    // add resolved status from v1: breach_resolved
+    // add resolved status from v1: breaches_resolved
     breach.IsResolved = !!resolvedBreachesV1.includes(breach.recencyIndex)
 
     // add resolved status from v2: breach_resolution
     if (breachResolutionV2[breach.recencyIndex] && !breach.IsResolved) {
-      const IsResolved = !breachResolutionV2[breach.recencyIndex].isActive
-      breach.IsResolved = breach.IsResolved || IsResolved
+      const IsResolved = breachResolutionV2[breach.recencyIndex].isResolved
+      breach.IsResolved = breach.IsResolved || IsResolved // if either v1 or v2 is marked as resolved, breach is resolved
```

Comment:
Could this loop be optimized or more readable with [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)?
i.e.
```
for (const breach of foundBreachesWithRecency) {
    // if either v1 or v2 is marked resolved, breach is resolved
    breach.IsResolved = !!resolvedBreachesV1.includes(breach.recencyIndex) ?? !!breachResolutionV2[breach.recencyIndex].isResolved
}
```

Reply:
This is cool, however, without the if checks, I think we might see a null pointer exception if
`breachResolutionV2[breach.recencyIndex]` does not exist

Reply:
we might be able to do: `!!breachResolutionV2[breach.recencyIndex]?.isResolved`

Reply:
The issue I have with ^ is that it's actually not as readable to me. I think the traditional if/else statements are more readable for engineers of all levels.

Reply:
Updated with `||`